sycl/source/detail/xpti_registry.hpp

```diff
@@ -95,6 +95,13 @@ class XPTIRegistry {
   static void bufferAssociateNotification(void *UserObj, void *MemObj);
   static void bufferReleaseNotification(void *UserObj, void *MemObj);
   static void bufferDestructorNotification(void *UserObj);
+  static void bufferAccessorNotification(void *UserObj, void *AccessorObj,
+                                         uint32_t Target, uint32_t Mode,
+                                         const detail::code_location &CodeLoc);
+  static xpti::trace_event_data_t *
```

Comment:
I believe, this should be guarded with `XPTI_ENABLE_INSTRUMENTATION`. Otherwise it breaks non-xpti builds.